src/App.css

```diff
@@ -2,37 +2,58 @@
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.Navbar li{
```

Comment:
Please create a folder named components under src and inside open a folder named Navbar put your navbar file inside and create a style file for navbar, move your styled to therr

Reply:
app-logo classına heigh:40vmin; verdıgımde bozuldu navbar